float64 x
float64 y
float64 cos
float64 sin
uint32 detections
uint32 id
uint16 status